app/views/courses/show.html.haml

```diff
@@ -27,21 +27,17 @@
         .row
           .col-md-8
```

Comment:
Can we try changing this to `.col-md-12`? It's not taking up the full width on desktop.